src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -339,6 +340,13 @@ namespace HeatBalanceManager {
 		// Get the heat balance input at the beginning of the simulation only
 		if ( ManageHeatBalanceGetInputFlag ) {
 			GetHeatBalanceInput(); // Obtains heat balance related parameters from input file
+
+			// Surface octree setup
+			//  The surface octree holds live references to surfaces so it must be updated
+			//   if in the future surfaces are altered after this point
+			surfaceOctree.init( DataSurfaces::Surface ); // Set up surface octree
```

Comment:
@DeadParrot I would suggest adding a check inside surfaceOctree.init to determine if any daylighting control objects are present.
```
if ( GetNumObjectsFound( "Daylighting:Controls" ) + GetNumObjectsFound( "Daylighting:DELight:Controls" )  > 0 ) 
```
will tell you if there are any daylighting objects present in the idf.  Sure, one could have these objects and not activate any control, but if these are not present, then you know for sure that daylighting is not active.  And you should probably check for any "Output:IlluminanceMap" objects as well - I don't remember if that can be used without any other daylighting controls present.